### DISCUSSION TITLE
Suggestion needed on solidus_braintree gem support in solidus 3.1.5(Latest)

### DISCUSSION THREAD

**Discussion by debarati20** [solidusio/solidus#4372] [Troubleshooting]
Upgrading the **solidus** version to latest **3.1.5**, causing issue for the deprecation of solidus_braintree gem. Any support or easy to fix options available for this?

Though I am aware of the supported version of **solidus_paypal_braintree** but wondering if any other workaround is available with solidus_braintree gem compatible with solidus v3.0.

**Comment by jarednorman** [comment]
Hey, just converted this to a Troubleshooting discussion since it's not really an issue that needs to be resolved with Solidus, so much as something that needs to get resolved with an extension.

Can you provide more information about the issue you're running into? What's the specific issue with *solidus_braintree* and the latest version of Solidus?

**Reply by debarati20** [reply]
Hi @jarednorman,
We are trying to upgrade solidus to 3.1.5 (Latest version) but as solidus_braintree is deprecated on 2018, we are trying to find out any replacement of solidus_braintree other than using solidus_paypal_braintree gem which will be compatible to solidus 3.1.5.

Is it possible to avoid using solidus_paypal_braintree in our case?
And is there any way to use solidus_braintree with solidus 3.1.5 without affecting solidus_braintree features?

**Reply by jarednorman** [reply]
My recommendation would be to switch to solidus_paypal_braintree, but if you didn't wish to do that, you could make some changes to the deprecated solidus_braintree to add Solidus 3.x support.

**Comment by kennyadsl** [comment]
Hey @debarati20 did you manage to solve this issue? If yes can you please mark an answer as the solution or post how you did it at the end?
